### ISSUE TITLE
No default settings

### ISSUE THREAD

**Issue by hfiguiere** [Unrud/djpdf#7]
I can't find where to set the default settings in the UI to keep the images are 100%. 

I also expected this to be the default.

**Comment by Unrud** [comment]
> I can't find where to set the default settings in the UI to keep the images are 100%.

The defaults can't be changed. I mainly use the CLI
```
flatpak run --filesystem=host --command=scans2pdf com.github.unrud.djpdf
```

**Comment by Unrud** [comment]
Fixed in bc29ab9dd03503f7e8eb80f334e7ed05db6a1509
